src/callbacks/evaluate_progress.cpp

```diff
@@ -56,6 +56,7 @@ EvalType evaluate(model& m, const std::string& metric_name, execution_mode mode)
   get_trainer().evaluate(&m, mode);
 
   // Get metric value
+  /*
   bool found_metric = false;
   EvalType metric_value = 0;
```

Comment:
```suggestion
  EvalType metric_value = -1.0;
  if (metric_name.size()) {
    bool found_metric = false;
```
Indent the rest as necessary.